assets/js/base/components/product-list/style.scss

```diff
@@ -40,10 +40,19 @@
 	position: relative;
 }
 
+.entry-content .wc-block-grid__product-image,
```

Comment:
Is this selector added for specificity? If that's the case, we might want to add a comment explaining it.